### ISSUE TITLE
Event enums don't export to Swift Correctly

### ISSUE THREAD

**Issue by jcampbell05** [firebase/geofire-objc#28]
Due to the fact NSEnum isn't used the Event Type enums export to swift as:

```swift
public struct GFEventType : RawRepresentable, Equatable {
    public init(_ rawValue: UInt32)
    public init(rawValue: UInt32)
    public var rawValue: UInt32
}
```

**Comment by nelsballs** [comment]
+1
Makes the GFQuery observers unusable for the moment

**Comment by jcampbell05** [comment]
This is the code to make it somewhat usable.

```swift
public extension GFQuery {
    
    //Enum for compatibility with Geofire's Obj-C Interface
    enum EventType: UInt32
    {
        case Entered = 0
        case Exited
        case Moved
    }
}
``

You then have to use it like so:
```swift
let type = GFEventType(rawValue: eventType.rawValue)

ref.observeEventType(type)
{
//Block code
}
``

Would be 200% better if we didn't need this work around :)

**Comment by jwngr** [comment]
@jdimond / @mcdonamp - Can either of you please provide some feedback here?

**Comment by iPhoNewsRO** [comment]
is also queryAtLocation affected by this? im trying the basic implementation but i get no response

**Comment by mcdonamp** [comment]
@iPhoNewsRO the issue here is that the enums aren't exported correctly in Swift--it shouldn't affect the API responses if the enums exist.

We're taking a look into exporting the correct enums using NSEnum--as always, PRs are always welcome :)

**Comment by iPhoNewsRO** [comment]
it compiles but doesn't return what i expect. probably i'll just need help implementing this

**Comment by mcdonamp** [comment]
Ok, then it's not related to this. [StackOverflow](http://stackoverflow.com/questions/tagged/geofire) or our [Google Group](https://groups.google.com/forum/#!forum/firebase-talk) might be good places to start looking for similar problems, and if you can't find anything posting there asking for help would good.

**Comment by mcdonamp** [comment]
Ok, should be solved in #29. Might be a few days before we merge it, and then get it out to Cocoapods. Since NS_ENUM is backwards compatible, I believe this will be 1.1.3.

**Comment by ricardosobral** [comment]
Would be useful to get either this asap in an interim or know when 1.1.3 is coming out. Tks. Quite unusable in iOS without these enums.

**Comment by jwngr** [comment]
Sorry about that! We will get 1.1.3 out on Monday.

**Comment by mcdonamp** [comment]
Hey folks--help up on the release with some CocoaPods issues, so it might still be a few days. That said, the code is merged and the v1.1.3 branch exists, so you can use:

```
pod "Geofire", :git => 'https://github.com/firebase/geofire-objc.git', :tag => 'v1.1.3'
```

Thanks!

**Comment by jwngr** [comment]
[Version `1.1.3`](https://cocoapods.org/pods/GeoFire) has been released to CocoaPods. Thanks for your patience. Let us know if you run into any issues with the latest bits!
